client/app/visualizations/chart/plotly/prepareLayout.js

```diff
@@ -118,10 +118,10 @@ function prepareBoxLayout(layout, options, data) {
 
 export default function prepareLayout(element, options, data) {
   const layout = {
-    margin: { l: 10, r: 10, b: 10, t: 25, pad: 4 },
+    margin: { l: 10, r: 10, b: 5, t: 20, pad: 4 },
```

Comment:
Without this change there are just bigger margins?

Reply:
Yes, and for me it looked too large; left/right paddings also changed a little bit, but in all cases it depends on axis labels (seems Plotly changed an algorithm that measures labels and now it preserves few more pixels for labels). Also seems that extra space also depends on chart size, therefore I was unable to fit margins perfectly, just as close as possible